proto/tink/ecdh_aead.proto

```diff
@@ -82,7 +82,7 @@ message EcdhAeadPublicKey {
 }
 
 // EcdhAeadPrivateKey represents ECDHDecrypt primitive.
-// key_type: type.hyperledger.org/hyperledger.aries.crypto.tink.EcdhAeadPrivateKey
+// key_type: type.hyperledger.org/hyperledger.aries.crypto.tink.EcdhAesAeadPrivateKey
 message EcdhAeadPrivateKey {
```

Comment:
Are you renaming these structures to include Aes?

Reply:
with the additional key managers that I need to add (P Curves + ChaCha and X25519 + AES) it doesn't make sense to create new proto but for keys, I will revert the proto bufs and use `EcdhAeadPublicKey ` and `EcdhAeadPrivateKey` in all the key managers

Reply:
the proto buf has been reverted. It's used for both NIST P curves and X25519 keys.